test/Ocelot.Benchmarks/AllTheThingsBenchmarks.cs

```diff
@@ -99,9 +99,9 @@ private void GivenOcelotIsRunning(string url)
                     logging.AddConfiguration(hostingContext.Configuration.GetSection("Logging"));
                 })
                 .UseIISIntegration()
-                .Configure(app =>
+                .Configure(async app =>
                 {
-                    app.UseOcelot().Wait();
+                    await app.UseOcelot();

```

Comment:
`WithUseOcelot` ?
I mean &rarr; `.Configure(WithUseOcelot)`

Reply:
The method to do that only exists at `Ocelot.AcceptanceTests.Steps`. Should I copy the method `WithUseOcelot` to `Ocelot.Benchmarks.AllTheThingsBenchmarks` (or even add reference to the `Ocelot.AcceptanceTests`) ? 

I can't see any benefit doing that. But I'm open to do if you prefer, just let me know.

<details><summary>Example as async</summary>

![image](https://github.com/user-attachments/assets/b0b88741-3d69-45bd-9626-d70d487fd068)

</details> 


<details><summary>Example as sync</summary>

Note: Using `.Wait()` or `.GetAwaiter().GetResult` 

![image](https://github.com/user-attachments/assets/c67e30bd-cdc6-493d-a774-2ab3470be9e7)


</details>

Reply:
Oh, sorry! This is benchmarks project. Not an issue!
Don't copy!
We have common testing project: [Ocelot.Testing](https://github.com/ThreeMammals/Ocelot/tree/develop/test/Ocelot.Testing)